docker_files/recipes/openfoam.core/meta/README.md

```diff
@@ -0,0 +1,11 @@
+# OpenFOAM Core 2.4.0
+This package provides core libraries, binaries and default configuration required
+by the OpenFOAM solvers.
+
+## Usage
+You only need to require this package and the libraries become available.
+In fact, the openfoam.core package is included in solvers so you shouldn't
+need to require this package manually.
+
+Note that libraries reside inside /openfoam directory so solvers will need
```

Comment:
Just checking: are you sure that libraries are inside `/openfoam`? Looking here, the libs are in `/usr/lib` while the `/openfoam` contains some OpenFOAM config files.